src/test/java/com/kodilla/ecommercee/domain/CartRepositoryTestSuite.java

```diff
@@ -0,0 +1,121 @@
+package com.kodilla.ecommercee.domain;
+
+import com.kodilla.ecommercee.repository.CartRepository;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@SpringBootTest
+@ExtendWith(SpringExtension.class)
+public class CartRepositoryTestSuite {
+    @Autowired
+    CartRepository repository;
+
+    List<CartProducts> cartProducts = new ArrayList<>();
```

Comment:
I believe that a separate list should be assigned to each Cart - I suggest adding new ArrayList<CartProducts> as the last parameter to each of the Carts below.

Reply:
Yes, you're right. This is my obvious mistake.